### PR TITLE
Fix user suggestion within the admin, fix #185

### LINKED ISSUE #185: Fix the user suggestion within the admin
As an administrator, I want to be able to preview users that I can add as a member to an organization.

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: js/components/organization/members.vue [hunk 1]
```diff
@@ -72,7 +72,7 @@
              v-repeat="member:org.members">
             <a class="pointer"
                  v-on="click: member_click(member)">
-                <img class="img-circle" alt="{{ _('User Image') }}"
+                <img class="img-circle" alt="{{ _('User Image') }}" width="60" height="60"
```
Comment (davidbgk) [inline js/components/organization/members.vue]: That modification is not part of the bug but it annoys me for months :)

Comment (noirbizarre) [inline js/components/organization/members.vue]: Be carefull, right now it scales to the display.
Have you tried with different display sizes ?

File: udata/core/user/api.py [hunk 2]
```diff
@@ -169,8 +169,7 @@ def get(self):
         return [
             {
                 'id': opt['text'],
-                'first_name': opt['payload']['first_name'],
-                'last_name': opt['payload']['last_name'],
+                'fullname': opt['payload']['fullname'],
```
Comment (noirbizarre) [inline udata/core/user/api.py]: It's intended to be the same fields as user or user reference.
It also match the fields from ``user/search.py``.

I think this is the ``js/components/form/user-completer.vue`` which need to be fixed.

File: js/components/organization/members.vue (hunk 1 above)
Comment (davidbgk) [inline js/components/organization/members.vue]: Indeed. I tried to fix that on the CSS side, not ideal but better than fixed.

**Comment by davidbgk** [comment]
@noirbizarre r?

**Comment by noirbizarre** [comment]
r+

**Pull request merged**